modules/decision_reviews/app/controllers/decision_reviews/v1/notice_of_disagreements_controller.rb

```diff
@@ -15,9 +17,10 @@ def show
       end
 
       def create
+        req_body_obj = normalize_area_code_for_lighthouse_schema(request_body_hash)
         nod_response_body = AppealSubmission.submit_nod(
           current_user: @current_user,
-          request_body_hash:,
+          request_body_hash: req_body_obj,
           decision_review_service:
         )
 
```

Comment:
The [spec file](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/decision_reviews/spec/requests/v1/notice_of_disagreements_spec.rb) for this controller is only at 80.95% test coverage. Please update the spec file.